app/code/community/TIG/MyParcelBE/controllers/CheckoutController.php

```diff
@@ -102,7 +103,7 @@ public function infoAction()
         $checkoutText['friday'] =                       $this->__('Friday');
         $checkoutText['saturday'] =                     $this->__('Saturday');
         $checkoutText['sunday'] =                       $this->__('Sunday');
-        $data['checkout_text'] = (object)$checkoutText;
```

Comment:
$checkoutText kan je ook veranderen in $text

Reply:
Done!